.github/workflows/artifact-publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish Java Package

```

Comment:
What will be the approx. artifact size and where we are using this ? As our ODE GitHub org is on free plan we have artifact storage limit of 500mb.

Reply:
The project's JAR file is about 57 MB in size. It is not currently being used, but it is for potential future use. It may be possible to reduce the size of the JAR file in the future.

Reply:
@dmccoystephenson seems like for Maven packages and other packages like npm, docker, etc  storage limit will be not applied for public repo's.  so I think we should be good to use artifacts. However for any private repo's this limits will apply. We have a backlog item for supporting GitHub artifacts setup in future release. Thanks!